src/ucs/Makefile.am

```diff
@@ -131,6 +131,8 @@ noinst_HEADERS = \
 	sys/iovec.h \
 	sys/iovec.inl \
 	sys/ptr_arith.h \
+	sys/ucs_netlink.h \
+	sys/ucs_rtnetlink.h \
```

Comment:
1. can we rename the H file to "netlink.h"? even though it's the same name as the system header i think it should be ok since they have diferent path
2. maybe unite these 2 headers / c files to one: netlink.h, netlink.c